### DISCUSSION TITLE
Release 2.1.0

### DISCUSSION THREAD

**Discussion by Voltstro** [Voltstro-Studios/UnityWebBrowser#282] [Announcements]
### Added

- Added support for allowing Unity to still build your project on unsupported UWB platforms
    - This doesn't mean that UWB will run on them!
- Added set/get zoom level
- Added open dev tools
- Added 'JS Methods'. A way of invoking .NET methods from JS.
- Added `OnClientInitialized` event
- Added `OnClientConnected` event

### Changed

- Bump deps
- Updated CEF to 121.3.13
- Upgrade Engine to .NET 8
- CEF Engine buffer improvements

### Fixed

- Fix Unity 2023 TMP Problems

<hr /><em>This discussion was created from the release <a href='https://github.com/Voltstro-Studios/UnityWebBrowser/releases/tag/2.1.0'>Release 2.1.0</a>.</em>

**Comment by RMichaelPickering** [comment]
Awesome! There are no instructions on how to upgrade from an older version yet. Uninstall first, or just update?

**Reply by Voltstro** [reply]
Update the UWB packages through the package manager like any other Unity package.

![image](https://github.com/Voltstro-Studios/UnityWebBrowser/assets/45032877/a4a76e14-4ae5-428a-86e6-8981df0c7133)
